### PR TITLE
Add validation for credit card owner birthday format

### PR DESCRIPTION
**Pull request by hugomaiavieira**
When doing this:

``` ruby
credit_card = MyMoip::CreditCard.new(owner_birthday: '80/01/1990')
```

I was getting an `ArgumentError - invalid date` error, but I want to be able to do:

``` ruby
credit_card = MyMoip::CreditCard.new(owner_birthday: '80/01/1990')
if credit_card.valid?
  # do something
else
  # do something else
end
```

### CODE REVIEW

File: lib/mymoip/credit_card.rb [hunk 1]
```diff
@@ -27,10 +28,9 @@ def logo=(value)
     end
 
     def owner_birthday=(value)
-      unless value.nil?
-        value = Date.parse(value.to_s)
-      end
-      @owner_birthday = value
+      value = Date.parse(value.to_s) unless value.nil?
+      rescue; ensure
```
Comment (Irio) [inline lib/mymoip/credit_card.rb]: Can you specify name exception name that you're rescuing?

**Comment by hugomaiavieira** [comment]
@Irio done.

File: test/lib/test_creditcard.rb [hunk 2]
```diff
@@ -58,6 +58,14 @@ def test_owner_birthday_accepts_string_version_of_dates
     assert_equal Date.new(1980, 12, 20), subject.owner_birthday
   end
 
+  def test_validate_format_of_birthday_date
```
Comment (Irio) [inline test/lib/test_creditcard.rb]: What about break this test into two? The first one to something like "Accepts input of invalid dates" with the `assert_equal '50/12/1980', subject.owner_birthday` assertion and "Invalid owner_birthdays invalidates the subject".

Comment (hugomaiavieira) [inline test/lib/test_creditcard.rb]: @Irio done

**Comment by Irio** [comment]
Nice. Thanks!

**Pull request merged**